pytorch_toolkit/object_detection/face_detection.md

```diff
@@ -103,66 +103,46 @@ If you would like to start **fine-tuning** from pre-trained weights do not forge
         python evaluation.py -p <OUTPUT_FOLDER> -g <WIDERFACE_MATLAB_ANNOTATION>
         ```
 
-### 6. Export to ONNX\*
+### 6. Export PyTorch\* model to the OpenVINO™ format
 
-* In most cases you can convert PyTorch\* model to the ONNX\* format by running the `export.py` script:
-
-    ```bash
-    python ../../external/mmdetection/tools/export.py \
-          $CONFIGURATION_FILE \
-          <CHECKPOINT> \
-          model.onnx
-    ```
-
-* If your model is SSD-like detector you can convert PyTorch\* model to the ONNX\* format by running the `export_ssd.py` script.
-
-    > **Note** the model exported in such way will produce a bit different results (non-significant in most cases) but it also might be faster that model exported by `export.py`. The `export.py` can export SSD models as well.
-
-    ```bash
-    python ../../external/mmdetection/tools/export_ssd.py \
-          $CONFIGURATION_FILE \
-          <CHECKPOINT> \
-          model.onnx
-    ```
-
-### 7. Convert ONNX\* model to the OpenVINO™ format
-
-Convert ONNX model to the OpenVINO™ format with the Model Optimizer with the command below:
+To convert PyTorch\* model to the OpenVINO™ IR format run the `export.py` script:
 
 ```bash
-python ../../external/mmdetection/tools/convert_to_ir.py \
-    $CONFIGURATION_FILE \
-    model.onnx \
-    <EXPORT_FOLDER>
+python ../../external/mmdetection/tools/export.py \
+      $CONFIGURATION_FILE \
+      <CHECKPOINT> \
+      <EXPORT_FOLDER> \
+      openvino
 ```
 
- This produces model `model.xml` and weights `model.bin` in single-precision floating-point format
- (FP32). The obtained model expects **normalized image** in planar BGR format.
+This produces model `model.xml` and weights `model.bin` in single-precision floating-point format
```

Comment:
not 'model.xml' in general case. The file name depends on name of configuration file. In my opinion this fact should be reflected in documentation or changed in mmdetection submodule.

Reply:
Nice catch. Fixed.